### PR TITLE
[new release] reparse and reparse-unix (2.0.0)

### PR DESCRIPTION
**Pull request by bikallem**
Recursive descent parsing library for ocaml

- Project page: <a href="https://github.com/lemaetech/reparse">https://github.com/lemaetech/reparse</a>

##### CHANGES:

- Rewrite the whole package to use exceptions rather than `result` type
- Adds many more parsing combinators
- Adds comprehensive unit tests
- Adds comprehensive documentation, host documentation and add links in repo home page
- Adds abstraction for input source
- Provides unix file source and string input source
- Adds separate package `reparse-unix` for unix file input
- Adds calc.ml and json.ml in examples.

### CODE REVIEW

**Comment by bikallem** [comment]
@mseri Hopefully, 2nd time lucky. 🤞

File: packages/reparse-unix/reparse-unix.2.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "Provides support for parsing files as source of input for reparse library "
+maintainer: ["Bikal Lem"]
+authors: ["Bikal Lem <gbikal@gmail.com>"]
+license: "MPL-2.0"
+homepage: "https://github.com/lemaetech/reparse"
+bug-reports: "https://github.com/lemaetech/reparse/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.10.0"}
+  "reparse" {= "2.0.0"}
```
Comment (mseri) [inline packages/reparse-unix/reparse-unix.2.0.0/opam:14]:
```suggestion
  "reparse" {= version}
```

Looks good, here you can use `version`, this way you can avoid updating it each time (no need to do anything for this release)

**Comment by mseri** [comment]
Feel free to announce as you did for http-cookie

**Comment by bikallem** [comment]
Thanks. Will do.

**Pull request merged**